packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-mutation-observer-ss/ss-mutation-observer-10k.benchmark.js

```diff
@@ -25,7 +25,7 @@ benchmark(`dom/synthetic-shadow/mutation-observer/10k`, () => {
         tableRows = [...tableElement.shadowRoot.querySelectorAll('benchmark-table-component-row')];
     });
 
-    run(async () => {
+    run(() => {
```

Comment:
Technically this does change the benchmark, due to the extra microtask, but it's probably not worth worrying about.